crates/matrix-sdk-crypto/src/backups/keys/backup.rs

```diff
@@ -93,6 +102,11 @@ impl MegolmV1BackupKey {
         self.inner.version.lock().unwrap().clone()
     }
 
+    /// Get the MAC key that is used with the backup key
+    pub fn mac_key(&self) -> &Option<HmacSha256Key> {
```

Comment:
`&Option<T>` is an anti-pattern we should use `Option<&T>`. You can use `Option::as_ref()` to achieve this.